tests/LINEBot/GetFollowerIdsTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Copyright 2016 LINE Corporation
```

Comment:
2020

Reply:
@moririnson 
thanks for reviews.
fixed.